common_features.mk

```diff
@@ -318,10 +318,6 @@ ifeq ($(strip $(TERMINAL_ENABLE)), yes)
     OPT_DEFS += -DUSER_PRINT
 endif
 
-ifeq ($(strip $(USB_HID_ENABLE)), yes)
```

Comment:
This had to be moved as `usb_hid.mk` seemed to be trashing the VPATH, its also protocol logic so more suited to `protocol.mk` anyway.